platform/platform-api/src/com/intellij/ui/jcef/JBCefScrollbarsHelper.java

```diff
@@ -233,7 +233,7 @@ private static int getThumbRadiusPx() {
     Color color = ObjectUtils.notNull(colorsScheme.getColor(key), key.getDefaultColor());
     double alpha = ObjectUtils.notNull(getScrollbarAlpha(key), color.getAlpha()) / 255.0;
 
-    return String.format(Locale.ROOT, "rgba(%d, %d, %d, %f)", color.getRed(), color.getBlue(), color.getBlue(), alpha);
+    return String.format(Locale.ROOT, "rgba(%d, %d, %d, %f)", color.getRed(), color.getBlue(), color.getGreen(), alpha);
```

Comment:
`rgba(...)` - so, probably, the first duplicate should be modified. 
Thanks for the hint!

Reply:
good catch. Fixing :)

Reply:
@mallowigi thank you for the fix!
Committed  29421e7